### PR TITLE
fix solution overlap & remove console log

### PR DESCRIPTION
**Pull request by skylarbarrera**

### CODE REVIEW

File: src/components/send/SendHeader.js [hunk 1]
```diff
@@ -133,7 +133,7 @@ export default function SendHeader({
 
   return (
     <Fragment>
-      {ios && <SheetHandle />}
+      <SheetHandle />
```
Comment (brunobar79) [inline src/components/send/SendHeader.js:136]: I already fixed this. Why am I seeing it again?

File: src/screens/SendSheet.js [hunk 2]
```diff
@@ -455,12 +454,7 @@ export default function SendSheet(props) {
   ]);
 
   return (
-    <Container
-      additionalTopPadding
-      as={android && SlackSheet}
-      contentHeight={sheetHeight}
-    >
-      {ios && <StatusBar barStyle="light-content" />}
+    <Container>
```
Comment (brunobar79) [inline src/screens/SendSheet.js:457]: @osdnk are u ok with this change?

Comment (skylarbarrera) [reply src/screens/SendSheet.js:457]: you both solved in different ways so when I pulled I had 2 handles, this seemed like the best solution and I tested on both OS

Comment (skylarbarrera) [reply src/screens/SendSheet.js:457]: status bar removal was actually an accident, above comment refers to your other

File: src/components/send/SendHeader.js (hunk 1 above)
Comment (skylarbarrera) [reply src/components/send/SendHeader.js:136]: you both solved in different ways so when I pulled I had 2 handles, this seemed like the best solution and I tested on both OS

Comment (osdnk) [reply src/components/send/SendHeader.js:136]: if it works, it works

File: src/screens/SendSheet.js (hunk 2 above)
Comment (osdnk) [reply src/screens/SendSheet.js:457]: Are you sure it works?

**Comment by osdnk** [comment]
it's good

**Review by osdnk — APPROVED** [review]

**Pull request merged**